data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+			`	They say they will be waiting for your return, and wish you good luck.`
+				accept
+	npc "scan outfits"
+		personality staying uninterested target
+		government "Quarg"
+		system "Enif"
+		fleet
+			names "quarg"
+			variant
+				"Quarg Skylark"
+				"Quarg Wardragon"
+	on complete
+		payment 200000
+		conversation
+			`Once again the engineers remove your Outfit Scanner, taking a copy of the scan logs with them, and handing another copy to the Heliarchs, who hand you <payment>.`
+			`	They return the outfit to your ship just like before, you still not being able to tell what they did to it.`
+			`	"Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, we will."`
+
+
+
+mission "Quarg Interrogation"
+	invisible
+	landing
+	source
+		government "Quarg"
+		not planet "Humanika"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 2-B: done"
+	on offer
+		conversation
+			`As you land on <origin>, a Quarg is patiently waiting right outside. Seeing that it won't go away, you head outside to speak with it.`
+			`	"Hello, <first> <last>," it says.`
+			choice
+				`	"Hi, can I help you?"`
+				`	"Is there some sort of problem?"`
+			`	It blinks, then looks to the side slowly, to a Wardragon parked farther away.`
+			`	"You were seen landing on our worlds and rings, bringing about strange scanning mechanisms, and were also found to be scanning some of our ships. Why is that?"`
+			choice
+				`	"I was just curious about your species."`
+					goto lie
+				`	"What are you talking about? I never did any of that. Must have been my cousin."`
+					goto lie
+				`	"Scanning? I'm sorry, I think my ship just had some malfunctions, I didn't mean to offend."`
+					goto lie
+				`	"The Heliarchs asked me to do it."`
+					goto truth
+			label lie
+			`	Around ten seconds pass without it saying anything, simply staring at your eyes and blinking slowly.`
+			`	"I see."`
+			`	Without another word, it walks away, headed to the Wardragon.`
+				decline
+			label truth
+			`	"And why did you agree? Do you wish to join their ranks, perhaps?" It asks.`
+			choice
+				`	"I just wanted to help, it's not like I destroyed your ships or something."`
+					goto justhelp
+				`	"They told me that the Quarg keep others in 'separate boxes', and that you hinder their development."`
+					goto hinder
+			label justhelp
+			`	"Perhaps not, but you agreed to collect intel on us and our worlds, which hold countless civilians, and deliver the findings to a group that hates us."`
+				goto end
+			label hinder
+			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai, or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
```

Comment:
```suggestion
			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
```
This seems a little aggressive for a Quarg. That might just be my own perception of the Quarg though, which I always saw as displaying little to no emotion. Then again, this story seems to be heading the direction of painting the Quarg in this fashion, so it might fit. Up to you.

Reply:
it is intentionally aggressive